validphys2/src/validphys/utils.py

```diff
@@ -7,6 +7,7 @@
 
 from frozendict import frozendict
 import numpy as np
+from reportengine.compat import yaml
```

Comment:
```suggestion
```
It' imported below.

Reply:
Thanks, addressed in [84c6ace](https://github.com/NNPDF/nnpdf/pull/1979/commits/84c6ace122b517379ba0c897f66f1a2fab8ed275)